### PR TITLE
JMkvpropedit: Add version 1.5.2

### PR DESCRIPTION
**Pull request by versedwildcat**
JMkvpropedit is a batch GUI for mkvpropedit (part of MKVToolNix) written in Java.
Requires Java 8 or newer.

Source and downloads at:
https://github.com/BrunoReX/jmkvpropedit

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/jmkvpropedit.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.5.2",
+    "description": "A batch GUI for mkvpropedit (part of MKVToolNix) written in Java",
+    "homepage": "https://github.com/BrunoReX/jmkvpropedit",
+    "license": "BSD 2-Clause \"Simplified\" License",
```
Comment (issaclin32) [inline bucket/jmkvpropedit.json:5]:
```suggestion
    "license": "BSD-2-Clause",

```
BSD 2-Clause "Simplified" License == BSD 2-Clause
See https://spdx.org/licenses/BSD-2-Clause.html

File: bucket/jmkvpropedit.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "suggest": {
+        "JDK": [
+            "java/openjdk",
+            "java/temurinjdk"
+        ]
+    },
+    "url": "https://github.com/BrunoReX/jmkvpropedit/releases/download/v1.5.2/jmkvpropedit-v1.5.2.zip",
+    "hash": "1a3998ee7ad343852b0f8e669278b03a88d546a7335dec166cead96c33fc27b9",
+    "pre_install": "$manifest.persist | ForEach-Object { if (-not (Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" | Out-Null } }",
```
Comment (rashil2000) [inline bucket/jmkvpropedit.json:14]: That's quite... clever. I didn't know you could access the `$manifest` object inside the manifest itself.

File: bucket/jmkvpropedit.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.5.2",
+    "description": "A batch GUI for mkvpropedit (part of MKVToolNix) written in Java",
+    "homepage": "https://github.com/BrunoReX/jmkvpropedit",
+    "license": "BSD-2-Clause",
+
```
Comment (rashil2000) [inline bucket/jmkvpropedit.json:6]: Remove empty line

**Review by rashil2000 — APPROVED** [review]

File: bucket/jmkvpropedit.json (hunk 2 above)
Comment (versedwildcat) [reply bucket/jmkvpropedit.json:14]: Yeah it's a quite interesting [feature](https://scoop-docs.vercel.app/docs/concepts/Pre--and-Post-install-scripts.html#variables). I first saw it used in the NirSoft bucket. (After dealing with the pain of trying to manually check if the file exists...)

**Pull request merged**